mobile/lib/features/wallet/share_invoice_screen.dart

```diff
@@ -58,6 +63,24 @@ class ShareInvoiceScreen extends StatelessWidget {
                 ),
               ]),
             ),
+            // Faucet button, only available if we are on regtest
+            Visibility(
+              visible: config.network == "regtest",
+              child: OutlinedButton(
+                onPressed: () {
+                  payInvoiceWithFaucet(invoice);
```

Comment:
:question: Are we not `await`ing here on purpose?

Reply:
spot on, it's a sync call - doing http requests async in dart requires a lot more wizardry, which is just not practical for a non-production code. 
at least it has a timeout by default.